packages/ui/src/components/va-modal/VaModal.vue

```diff
@@ -4,9 +4,18 @@
       <div class="va-modal" v-if="valueComputed">
         <div
           class="va-modal__overlay"
+          v-if="$props.overlay"
           :style="$props.overlay && computedOverlayStyles"
           @click="onOutsideClick"
         />
+
+        <div
+          class="va-modal__overlay-blur"
+          v-if="$props.blur"
+          :style="$props.blur && computedBlurStyles"
+          @click="onOutsideClick"
+        />
```

Comment:
Can we remove that and only use `class="va-modal__overlay"` (div above) with blurry styling if the `blur` property is on?